Steamworks/src/org/usfirst/frc/team4786/robot/subsystems/DriveTrain.java

```diff
@@ -43,9 +71,58 @@ public void brake(){
 	//Drive Command for Open Loop System;
 	//Should be obsolete once PID is Implemented
 	public void openLoopDrive(double leftInput, double rightInput) {
+		//Change Talon modes to PercentVbus
+		frontLeft.changeControlMode(TalonControlMode.PercentVbus);
+		frontRight.changeControlMode(TalonControlMode.PercentVbus);
+		
 		double leftOutput = leftInput * RobotMap.openLoopSpeedScaling;
 		double rightOutput = rightInput * RobotMap.openLoopSpeedScaling;
 		frontLeft.set(leftOutput);
 		frontRight.set(rightOutput);
 	}
+	
+	//Begin PID Functions
+	
+	public void driveToPosition(double distanceToDrive){
+		//Run convertToRotations function
+		double rot = convertToRotations(distanceToDrive);
+		
+		//Change Talon modes to "position" just in case
+    	//they were in another mode before
+    	frontLeft.changeControlMode(TalonControlMode.Position);
+    	frontRight.changeControlMode(TalonControlMode.Position);
+		
+		//Make motors drive number of rotations
+		//calculated before by convertToRotations()
+		frontLeft.set(rot);
+		//Make sure we inverse this right side,
+		//otherwise, you have a spinning robot on your hands
+		frontRight.set(-rot);
+		
+		SmartDashboard.putNumber("Position", frontLeft.getPosition());
+	}
+	
+	//Have these functions to get CANTalon objects
+	//since they are private
+	public CANTalon getFrontLeft(){
```

Comment:
You should remove these accessor functions. You should try to keep the talon srx motors as purely an implementation detail of the subsystem and not exposed to other code.